walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/util/controlflow/Result.kt

```diff
@@ -11,7 +11,7 @@ import kotlin.Result as KotlinResult
 
 typealias Success = Boolean
 
-internal sealed class Result<out S> {
```

Comment:
Is this used outside of Wallet Library? The idea was to use this internally and expose VerifiedIdResult outside Wallet Library which native result.